### ISSUE TITLE
Keep score

### ISSUE THREAD

**Issue by mynamewastaken** [madSimonJ/ReBourne#1]
An idea for adding an additional feature (would handle some more user input as well as reading/writing/parsing files): keep user score in the form of name + score, add an option to view high scores, persist to disk in some format (or use json/yaml/etc).

**Comment by madSimonJ** [comment]
That's a great idea.  My brief at the moment is to try and re-create the original Osbourne code as closely as I can, so that would be out of scope for now.  I'm open to coming back another time & doing it, though - unless you'd like to open a Pull Request?

**Comment by mynamewastaken** [comment]
Nah, go for it. You did mention you were trying to learn PowerShell 👍
